packages/components/psammead-topic-tags/src/index.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { string, element, oneOf, shape, arrayOf } from 'prop-types';
+import styled from '@emotion/styled';
+import { C_LUNAR, C_EBON, C_METAL } from '@bbc/psammead-styles/colours';
+import { GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { getBrevier } from '@bbc/gel-foundations/typography';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+
+const ltrRtl = (ltrValue, rtlValue) => ({ dir }) =>
+  dir === 'ltr' ? ltrValue : rtlValue;
```

Comment:
I realise this kind of thing is all over out codebase, but it is an anti-pattern. The more dynamic our styles, the more of a performance hit we get as they cannot be cached.

Please can we explore using symmetrical margins? Being able to draw a line down the middle of a design and flip it without the styles needing to change is a real sweet spot.